README.md

```diff
@@ -0,0 +1,130 @@
+# Tahoe-LAFS Infrastructure
+
+The software that defines the Tahoe-LAFS project's infrastructure.
+
+## Abstract
+
+This repository was initially created in Sep 2022 as an effort to deploy and
+manage a `testgrid` server using NixOS (now retired).
+
+Since January 2025, some members of the community are seeking to re-use this repository
+to deploy and manage some new systems:
+
+- a `testgrid` server re-using the NixOS configuration written in 2022
+- a `webforge` server to [MoveOffTrac](https://github.com/tahoe-lafs/MoveOffTrac)
+
+In addition to the [Nix](https://nixos.org/) code allowing to manage the software installed on those servers,
+an [OpenToFu]() plan should handle the definition of the related infrastructure (e.g.: VMs and DNS records).
+
+## ToDo/WiP
+
+The content of this repository is mostly a work in progress (as time of writing).
+
+Here follows a list of things to do, if not already in progress (see open issues).
+
+For [Testgrid](#testgrid):
+
+- [X] Fix and rework the Nix code to deploy the current Tahoe-LAFS software
+- [ ] Implement or adapt the OpenToFu plan to cover the `testgrid` and
+  import the resource already deployed (on Hetzner and Gandi if possible)
+- [ ] Refactor the Nix code to integrate the `nixosConfiguration` with the top-level flake,
+  which will be used to manage and automated the deployment of all NixOS configurations
+
+For [Webforge](#webforge):
+
+- [ ] Implement the OpenToFu plan for Hetzner and its CI/CD workflow with GHA
+- [ ] Implement the NixOS configuration and its CI/CD workflow with GHA
+
+For the [Deployment Workflow](#deployment-workflow):
+
+- [ ] Automated the DNS configuration using OpenToFu and the relevant credentials from Gandi
+- [ ] Describe the [provisioning](#provisioning) steps with the relevant paths and snippets
+- [ ] Investigate how to allow pull request from (external) fork (with read-only token?)
+
+## How to Contribute
+
+- Read the doc and the code, consult the existing issues and ask questions (via `#tahoe-lafs` on [Libera](https://libera.chat/))
+- Describe the bug/problem or the missing info/feature in a new issue in this repository
+- Create a new branch (requires **Write** permissions in this repository) and start hacking
```

Comment:
Please don't do this, it would stall progress.
This is GitHub, anyone can clone to create a PR.

Reply:
As explained [here](https://github.com/tahoe-lafs/infrastructure/pull/20#discussion_r1925650430), anyone can fork/clone and PR, but it will not work w/o us solving a problem for which I have not good solution yet.